schemas/private/ekscluster-kfd-v1alpha2.json

```diff
@@ -1458,6 +1458,9 @@
         },
         "vpcId": {
           "$ref": "#/$defs/Types.AwsVpcId"
+        },
+        "bucketNamePrefix": {
+          "$ref": "#/$defs/Types.AwsS3BucketName"
```

Comment:
You should create a new API version because the v1alpha2 has already been released.